src/matrix/room/timeline/Timeline.js

```diff
@@ -304,6 +423,19 @@ export class Timeline {
         return null;
     }
 
+    getFromAllEntriesById(eventId) {
+        if (!this._allEntries) {
+            // if allEntries isn't loaded yet, fallback to looking only in remoteEntries
+            return this.getByEventId(eventId);
+        }
+        for (const entry of this._allEntries) {
```

Comment:
Given that we won't allow replying to unsent events, should we revert this to be just looking in remoteEntries now?